python/doc/examples/data_analysis/statistical_tests/plot_kolmogorov_distribution.py

```diff
@@ -5,18 +5,18 @@
 # %% 
 
 # %%
-# In this example, we draw the Kolmogorov-Smirnov distribution for a sample size 10. We want to test the hypothesis that this sample has the `Uniform(0,1)` distribution. The K.S. distribution is first plot in the case where the parameters of the Uniform distribution are known. Then we plot the distribution when the parameters of the Uniform distribution are estimated from the sample.
+# In this example, we draw the Kolmogorov-Smirnov distribution for a sample size 10. 
+# We want to test the hypothesis that this sample has the `Uniform(0, 1)` 
+# distribution. 
+# The K.S. distribution is first plot in the case where the 
+# parameters of the Uniform distribution are known. 
+# Then we plot the distribution when the parameters of the `Uniform` 
```

Comment:
```suggestion
# parameters of the uniform distribution are known.
# Then we plot the distribution when the parameters of the uniform
```
The formatting of 'uniform' seems a little random: sometimes lower-case u, sometimes upper-case U, sometimes as code, sometimes as plain text... In order to be consistent with other doc files, I think it is best to write "uniform" as plain text with lower-case u.

Reply:
You are right.